lib/periodic_time_range.rb

```diff
@@ -4,32 +4,20 @@
 require 'active_support'
 require 'active_support/time'
 
-require 'periodic_time_range/version'
+require_relative 'periodic_time_range/version'
 
 module PeriodicTimeRange
-
   FIFTEEN_MINUTES = 15.minutes.freeze
 
-  def self.to_range(current_time, duration_in_secs)
-    candidate_ranges(current_time, duration_in_secs).find { |r|
-      r.cover?(current_time - duration_in_secs)
-    }
+  def self.to_range(time, duration)
+    time = Time.zone.at(time.to_i)
```

Comment:
Once the API is on ruby 2.7 or above, we can change this to `time = time.floor`.

https://blog.saeloun.com/2019/08/20/ruby-2-7-time-ceil-and-time-floor-methods.html

It looks like we can also [set a minimum ruby version](https://guides.rubygems.org/specification-reference/#required_ruby_version=) for the gem in our gemspec file.